libkbfs/keybase_daemon_rpc.go

```diff
@@ -333,6 +336,9 @@ func (k *KeybaseDaemonRPC) OnConnect(ctx context.Context,
 		return err
 	}
 
+	// Set k.server only if err == nil.
+	k.server = server
```

Comment:
Should we set it to `nil` in `OnDisconnected`?  I don't think it would really affect things, but probably good practice in case we add protocols later that don't get added until after startup.